### ISSUE TITLE
get http headers ?

### ISSUE THREAD

**Issue by saeedvir** [ixudra/curl#76]
```
$c = \Curl::to($url)
->returnResponseObject()
->withHeader('User-Agent:Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36')

->withOption('HEADER',1);

```

http headers  and page content are together : 

```
HTTP/1.1 200 OK\r\n
Cache-Control: no-cache, private\r\n
Content-Type: text/html; charset=UTF-8\r\n
Set-Cookie: XSRF-TOKEN=eyJpdiI6IllLYlY4a3R6clhwNmMySmVTOERuUmc9PSIsInZhbHVlIjoicWxrVGdqbDBWSVpGY1J4NkVzbEU2XC9YaVRqRkJkVjdXKytUeFhaT0Q2V0hRRno4dUk2NWpuNFZUOTRYb ▶
Set-Cookie: laravel_session=eyJpdiI6Ik1zR3crXC9HVmxpV3hOSlg3Sm1PRFh3PT0iLCJ2YWx1ZSI6IkNMUXZxR0dINkV3S3NNNnpjM2pacDVsWW9TTVNreFwveU1NUVwvbDhhUlgySjJsVUtxOEswbEJU ▶
Transfer-Encoding: chunked\r\n
Date: Fri, 17 Nov 2017 04:40:29 GMT\r\n
Accept-Ranges: bytes\r\n
Server: LiteSpeed\r\n
Connection: close\r\n
\r\n
<!DOCTYPE HTML>\r\n
<html xmlns="http://www.w3.org/1999/xhtml" lang="fa" dir="rtl">\r\n
<head>\r\n
...
..
.
.
.
.
.
```

for ex : 
```
$c->content
```
and
```
$c->headers
```

**Comment by elimentz** [comment]
I'm sorry, I have absolutely no clue what you are talking about...

**Comment by elimentz** [comment]
Any chance you could explain a bit more? Otherwise I will close this ticket

**Comment by elimentz** [comment]
Now available in v6.16.0 (I think)
